### PR TITLE
Public documentation for Accept-Language header

### PR DESCRIPTION
**Pull request by Pchelolo**
WIP until we enable it for everything.

cc @wikimedia/services

### CODE REVIEW

File: v1/content.yaml [hunk 1]
```diff
@@ -152,6 +152,13 @@ paths:
             To get a 200 response instead, supply `false` to the `redirect` parameter.
           type: boolean
           required: false
+        - name: Accept-Language
+          in: header
+          description: >
+            The desired language variant code for wikis where LanguageConverter is enabled.
+            Example: `sr-el` for latin trascription of the Serbian language.
```
Comment (jdforrester) [inline v1/content.yaml]: * Latin, * transcription

File: v1/content.yaml [hunk 2]
```diff
@@ -438,6 +445,13 @@ paths:
...
```
Comment (jdforrester) [inline v1/content.yaml]: Same here and below.

File: v1/content.yaml (hunk 1 above)
Comment (Pchelolo) [reply v1/content.yaml]: Sorry, didn't quite understand what you meant here

Comment (berndsi) [reply v1/content.yaml]: "Latin" should be upper case. "trascription" is a typo, missing an n. Should be "transcription".

Comment (Pchelolo) [reply v1/content.yaml]: oh.. didn't notice the typo, so the `*` made me confused. Sorry.

Comment (jdforrester) [reply v1/content.yaml]: My fault, I should have been clearer, sorry. :-)

**Pull request merged**